TelepresenceBot/core/src/main/java/com/novoda/tpbot/support/Observable.java

```diff
@@ -15,7 +13,7 @@ protected Observable() {
 
     public synchronized Observable<T> attach(Observer<T> observer) {
         if (observer == null) {
-            throw new DeveloperError("You cannot attach a null observer");
+            throw new NullPointerException("You cannot attach a null observer");
```

Comment:
Didn't have access to utils here and didn't want to add it as it isn't really necessary.